PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutAccount.java

```diff
@@ -121,28 +123,34 @@ void setIntent(String intent) {
     }
 
     /**
+     * Deprecated. Use PayPal module instead.
      * Grabs the client meta data id
      *
      * @return clientMetadataId
      */
+    @Deprecated
```

Comment:
For simplicity maybe we should just mark the classes as deprecated at the top level rather than every property and method. There may be some method invocations where merchants don't see the warning, but overall their integration should show them that the module is deprecated if each class is deprecated at the top level.  What do others think?

Reply:
I know in iOS, we did mark every public function, properties as well as the classes. I'm good with just top level deprecation line if that is enough for merchants. 

I know I ran into some test failures without some annotations in protocols, not sure what happens in Android.

Reply:
If it works we should do class level only yeah. That makes the most sense. My guess is Android Studio will cross out references e.g. ~client~.tokenize().